README.md

```diff
@@ -64,6 +64,10 @@ new_version.save
 directory.versions['v2'].file_names
 # => ["ocfl.rbs"]
 
+# Or on the head version
+directory.reopen_head_version.file_names
```

Comment:
I don't understand. Do you have to reopen the head version to get its file names?